wordle-web/app.vue

```diff
@@ -6,14 +6,49 @@
           <v-icon color="secondary"> mdi-book </v-icon>
         </template>
 
-        <v-app-bar-title>WordleApp</v-app-bar-title>
+        <v-app-bar-title></v-app-bar-title>
 
         <v-btn @click="router.push('/')">Home</v-btn>
-        <v-btn @click="router.push('/test')">Test</v-btn>
-        <v-btn icon="mdi-theme-light-dark" @click="toggleTheme" />
+        <v-btn @click="router.push('/wordOfTheDay')">WOTD</v-btn>
+        <!--<v-btn icon="mdi-theme-light-dark" @click="toggleTheme" />
         <v-btn icon="mdi-help-circle" @click="showHelpDialog = true" />
-        <HelpDialog v-model="showHelpDialog" />
+        <HelpDialog v-model="showHelpDialog" />   -->
+        <v-app-bar-nav-icon
+            color="secondary"
+            @click="toggleMenu"
+            :class="{ 'rotate': menuOpen}"
+          ></v-app-bar-nav-icon>
       </v-app-bar>
+
+      <v-navigation-drawer
+        location="right" 
+        v-model="menuOpen" app>
+        <v-list>
+          <v-list-item link @click="router.push('/')">
+            <v-list-item-title>Home</v-list-item-title>
+          </v-list-item>
+          <v-list-item link @click="router.push('/wordOfTheDay')">
+            <v-list-item-title>WOTD</v-list-item-title>
+          </v-list-item>
+          <v-list-item link @click="router.push('/instructions')">
+            <v-list-item-title>Instructions</v-list-item-title>
+          </v-list-item>
+          <!--<v-list-item>
+            <v-btn @click="showHelpDialog = true">Help</v-btn>
+            <HelpDialog v-model="showHelpDialog"/>
+          </v-list-item>
+          <v-list-item>
+            <v-btn icon="mdi-theme-light-dark" @click="toggleTheme" />
+</v-list-item> -->
+          <v-spacer></v-spacer>
+        </v-list>
+        <v-card-actions class="mt-auto">
```

Comment:
I think this could be removed now that you have instructions :) If you disagree, I would make the LetterResults responsive in the Help dialog

Reply:
Good point, I thought I may as well have it so that you don't have to go from the game to the instruction page to view it.